### PR TITLE
feat: Added syncAllGroups

### PR DESCRIPTION
**Pull request by alexrisch**
Added syncAllGroups call
Closes https://github.com/ephemeraHQ/converse-app/issues/543

### CODE REVIEW

File: i18n/translations/en.ts [hunk 1]
```diff
@@ -46,6 +46,17 @@ const en = {
...
+  common_activity: "COMMON ACTIVITY",
+  social: "SOCIAL",
+  address: "ADDRESS",
+  youre_the_og: "YOU'RE THE OG",
+  app_version: "APP VERSION",
+  security: "SECURITY",
+  revoke_other_installations: "Revoke other installations",
+  revoke_all_other_installations: "Revoke all other installations",
+  revoke_description:
+    "Revoke all other group installations of XMTP on other applications and devices.",
```
Comment (nplasterer) [inline i18n/translations/en.ts:59]:
```suggestion
    "Revoke all other V3 installations of XMTP on other applications and devices.",
```

File: screens/Profile.tsx [hunk 2]
```diff
@@ -330,6 +331,38 @@ export default function ProfileScreen({
     );
   }, [colorScheme, logout]);
 
+  const showRevokeActionSheet = useCallback(async () => {
```
Comment (nplasterer) [inline screens/Profile.tsx:334]: Totally doesn't have to happen in this PR but might make sense to show the number of other installations. Like a `revoke 10 other installations`.

Comment (nplasterer) [reply screens/Profile.tsx:334]: This can be retrieved from the new `InboxState`

File: utils/xmtpRN/conversations.ts [hunk 3]
```diff
@@ -296,6 +296,7 @@ const listConversations = async (client: ConverseXmtpClientType) => {
 const listGroups = async (client: ConverseXmtpClientType) => {
   // @todo => this will be adapted once we have a syncAllGroups method
   const groups = await fetchGroupsQuery(client.address);
+  // client.conversations.syncAllGroups();
```
Comment (nmalzieu) [inline utils/xmtpRN/conversations.ts:299]: goal of using this line is to remove the loop right after

**Review by nmalzieu — CHANGES_REQUESTED** [review]

**Review by nplasterer — APPROVED** [review]

**Pull request merged**